com.vrcfury.vrcfury/Runtime/VF/Component/VRCFuryHapticSocket.cs

```diff
@@ -82,6 +83,9 @@ public class DepthAction {
             if (fromVersion < 7) {
                 enableActiveAnimation = activeActions.actions.Count > 0;
             }
+            if (fromVersion < 8) {

```

Comment:
This shouldn't be here. If the legacy components should inherit the new default value, there's no need for an upgrade.